airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -129,7 +129,7 @@ def _render_log_id(self, ti: TaskInstance, try_number: int) -> str:
         return self.log_id_template.format(
             dag_id=ti.dag_id,
             task_id=ti.task_id,
-            run_id=ti.run_id,
+            run_id=getattr(ti, "run_id", ""),
```

Comment:
Does this work with an empty string?

cc @jedcunningham @uranusjr

Reply:
@jedcunningham @uranusjr  :pray:  - I'd love to re-release providers asap as the ones we have install gitpython and wheel due to my sloppines :(

Reply:
Hey @jedcunningham @uranusjr @kaxil -> I'd really love to merge it and re-release the providers soon. The "extra packages" in the last list need to go away and the cncf.kuberbnetes starts to be a problem for users of Airflow 2.1 (likely this one https://apache-airflow.slack.com/archives/CCV3FV9KL/p1647936437392429)

Reply:
Approved the PR but I still don't know about this and haven't dug deep

Reply:
Well. It certainl won't be worse than crashing when you try to send an elasticsearch log :)

Reply:
This is what 2.1 users would experience if they run this provider version.